Source/ThirdParty/GltfPipeline/processPbrMetallicRoughness.js

```diff
@@ -493,6 +513,9 @@ define([
         } else {
             if (defined(parameterValues.baseColorFactor)) {
                 fragmentShader += '    vec4 baseColorWithAlpha = u_baseColorFactor;\n';
+            } else if (!hasNormals && (defined(parameterValues.emissiveFactor) || defined(parameterValues.emissiveTexture))) {
+                // Don't use the default base color of (1,1,1) if the material is emissive only. This handling will be better once we support KHR_materials_unlit.
+                fragmentShader += '    vec4 baseColorWithAlpha = vec4(vec3(0.0), 1.0);\n';
```

Comment:
This breaks glTF spec, and I don't think we have any need to do it in code here.  The models can and should simply specify `baseColorFactor: [0, 0, 0, 1]` so that all engines know to turn this off.

Reply:
Yeah that is the better thing to do. Fixed.